packages/graph/presets/all.ts

```diff
@@ -21,6 +21,7 @@ import "../lists/index.js";
 import "../mail/index.js";
 import "../members/index.js";
 import "../onenote/index.js";
+import "../open-extensions/index.js";
```

Comment:
open extensions don't exist yet. we can merge this - but the api support isn't there yet.